lib/rbs/environment_loader.rb

```diff
@@ -34,16 +34,32 @@ def initialize(core_root: DEFAULT_CORE_ROOT, repository: Repository.new)
       @core_root = core_root
       @repository = repository
 
-      @libs = []
+      @libs = Set.new
```

Comment:
It is necessary to avoid an infinite loop.


I know EnvironmentLoader also checks duplication in `each_decl`, but this change is also necessary to avoid an infinite loop during `add` phase.